### PR TITLE
Desktop feature

### PR DESCRIPTION
**Pull request by amanuelbelete95**
**List of activities**

-  Used media queries in my CSS (Desktop: 768px or wider)
-  Used Grid and Flexbox to lay out items.
-  The desktop version of the portfolio is similar to the Figma template provided
-  Checked linter errors for CSS and HTML file
-  Used correct GitHub flow

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,439 +1,703 @@
...
+    padding: 10px 12px;
   }
 
-  .contact-section {
-    padding-top: 80px;
+  .about-section {
+    background-image: url("./cardsimage/aboutFrame.png");
+    background-repeat: no-repeat;
+    background-position: 740px 68px;
   }
```
Comment (levy002) [inline style.css]: - [ ] Kindly make sure the background image is visible on the `About Myself` section on the desktop version. :wink: 

   - ### YOUR DESIGN
   
      image

   - ### FIGMA DESIGN
   
      image

File: style.css [hunk 2]
```diff
@@ -1,439 +1,703 @@
...
-    flex-direction: column;
-    background-image: url("./Images/bg1.png");
-    background-repeat: no-repeat;
-    background-size: cover;
+    grid-template-columns: auto auto auto;
+    margin-top: 24px;
+    margin-left: 74px;
+    margin-right: 74px;
+    gap: 20px 10px;
   }
```
Comment (levy002) [inline style.css:583]: - [ ] Kindly make sure your project cards are visible on the desktop version.

   - ### YOUR DESIGN
   
     image

   - ### FIGMA DESIGN
   
     image

File: style.css [hunk 3]
```diff
@@ -1,439 +1,703 @@
...
+    display: flex;
+    justify-content: space-around;
+    gap: 40px;
+    height: auto;
+    padding-top: 200px;
+    background-image: url("./desktop.png");
+    background-repeat: no-repeat;
+    background-position: top right;
+    background-color: #1c1a19;
   }
```
Comment (levy002) [inline style.css]: - [ ] Kindly make sure to align the headline content correctly and make sure the background image is visible on the mobile version.

   - ### YOUR DESIGN
   
    image

   - ### FIGMA DESIGN
   
     image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @amanuelbelete95   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 4]
```diff
@@ -1,439 +1,703 @@
...
-    flex-direction: column;
-    background-image: url("./Images/bg1.png");
-    background-repeat: no-repeat;
-    background-size: cover;
+    grid-template-columns: auto auto auto;
+    margin-top: 24px;
+    margin-left: 74px;
+    margin-right: 74px;
+    gap: 20px 10px;
   }
```
Comment (emmiiorji) [inline style.css:583]: You have done great by making the project cards show up on the desktop version as requested by the previous reviewer. Kindly ensure that the design of the cards follow the figma design.

The card design with a cursor over it and the "see project" button shows how each of the cards should appear when they're hovered over. The remaining cards show how the cards should normally look.

Card when hovered:
image

Normal display of card:
image

File: index.html [hunk 5]
```diff
@@ -60,6 +70,8 @@ <h3 class="heading3">I am Software Engineer.</h3>
     <section class="work-section">
       <div class="work">
         <h1 class="work-title">My Recent Works</h1>
+        <hr class="desktop-hr" />
+
         <img src="./Images/mytitle image.png" class="work-image" />
         <h3 class="story-title">Multi-Post Stories</h3>
```
Comment (emmiiorji) [inline index.html:76]: [Optional] I recommend that you design the leading project as close to the design as possible. The image as shown below occupies a larger with than the description.

Your design:
image

Figma design:
image

File: index.html [hunk 6]
```diff
@@ -25,16 +25,26 @@
...
-        <p class="myinfo">
-          I can help you build a product , feature or website Look through some
-          of my work and experience! If you like what you see and have a project
-          you need coded, don’t hestiate to contact me.
-        </p>
+        <div class="div1">
+          <h1 class="heading1">Hey There.<br />I’m Amanuel</h1>
+          <h3 class="heading3">I am Software Engineer.</h3>
+          <p class="myinfo">
+            I can help you build a product , feature or website Look through
```
Comment (emmiiorji) [inline index.html:43]: [Optional] I suggest that you remove the horizontal overflow on the page in order to give a better user experience on the website.

Your design:
image

Figma design:
image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @amanuelbelete95, 

# Status: Changes Required ♻️

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

A little bit more

 ## Good Highlights✔️

- Linter checks are passing
- Proper use of GitHub flow
- Good commit messages

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AlphaNtihinduka — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
## Hi @amanuelbelete95 🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 

approved-accepted

Congratulations! 🎉 
TO HIGHLIGHT:

- Your project is professional :heavy_check_mark: 
- Your repo is professional :heavy_check_mark: 
- Well-structured files :heavy_check_mark: 
- Professional README.md file 💯
- No linter errors :heavy_check_mark: 

Now you can merge this branch and move on :1st_place_medal: 


## Optional suggestions
- ❗ I have noticed that you have not applied the optional changes requested by the previous reviewer about the horizontal scroll effect. Kindly ensure to apply them to have a professional project that suits recruiters' needs.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :four_leaf_clover:

**Pull request merged**